content/fast/using-lighthouse-bot-to-set-a-performance-budget/index.md

```diff
@@ -0,0 +1,295 @@
+---
+page_type: guide
+title: Using Lighthouse Bot to set a performance budget
+author: mihajlija
+description: |
+  You’ve done hard work to get fast, now make sure you stay fast by automating
+  performance testing in Travis CI with Lighthouse Bot.
+web_lighthouse: N/A
+web_updated_on: 2018-12-06
```

Comment:
oh we should probably update this to be today's date.

Reply:
It occurred to me but I wasn't sure if you do that automatically on the actual date of pushing to production. Anyways, it's fixed :)

Reply:
I'm looking into ways to automate it but it's still manual for now :P